.tools/readmes/scanner.py

```diff
@@ -144,7 +144,10 @@ def snippet(self, example, sdk_ver, readme_folder, api_name):
                         elif "snippet_files" in excerpt:
                             snippet_files = excerpt["snippet_files"]
                             # TODO: Find the best (or all?) snippet files, not the first.
-                            tag_path = snippet_files[0]
+                            full_path = snippet_files[0]
```

Comment:
These changes make assumptions about the folder structure which are not valid for all languages.
The changes assume the service folder ReadMe is in a folder 3 deep `*/*/*/README.md.
Execute the following command from the repo base.

`ls */*/README.*`

This does not apply for gov2, dotnetv3 or aws-cli.
Also, the naming for "cross-services" is not standardized. It is named "cross-service" in dotnetv3.

Reply:
@meyertst-aws understood in general, thus the "TODO". At this time, only Rust and JSv3 are using snippet_files, so I'm inclined to accept the PR as is and resolve this "imminently" by replacing scanner.py with a more robust implementation using the new awsdocs/aws-doc-sdk-examples-tools repo & library.